config/getting-started-xmonad.hs

```diff
@@ -0,0 +1,48 @@
+import System.Exit
+import XMonad
+import XMonad.Hooks.DynamicLog
+import XMonad.Hooks.ManageDocks
+import XMonad.Hooks.ManageHelpers
+import XMonad.Layout.Fullscreen
+import XMonad.Layout.LayoutHints
+import XMonad.Layout.NoBorders
+import XMonad.Prompt
+import XMonad.Prompt.ConfirmPrompt
+import XMonad.Prompt.Shell
+import XMonad.Util.EZConfig
+import XMonad.Util.Run (hPutStrLn, spawnPipe)
+
+main = do
+  xmproc <- spawnPipe "xmobar ~/.xmonad/xmobarrc -i ~/.xmonad/icons"
+  xmonad $
+    defaultConfig
+    { terminal = "xterm"
+    , manageHook =
+        myManageHook <+> fullscreenManageHook <+> manageDocks <+>
+            manageHook defaultConfig
+    , layoutHook =
+        fullscreenFloat $ fullscreenFocus $ layoutHints $ smartBorders $
+            avoidStruts $ layoutHook defaultConfig
```

Comment:
1. Are all of these layout modifiers still recommended?

2. Change `defaultConfig` to `desktopConfig`